client/signer_test.go

```diff
@@ -237,13 +252,9 @@ func TestSignPSS(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			// Verify with expected salt length.
-			err = rsa.VerifyPSS(signer.Public().(*rsa.PublicKey), k.opts.HashFunc(), digest[:], sig, &rsa.PSSOptions{SaltLength: k.saltLen, Hash: k.opts.HashFunc()})
-			if err != nil {
-				t.Error(err)
-			}
 
-			// Verify with default salt length.
+			// Different implementations may specify different salt length. Some have "keyBytes - digestSize - 2", some have
+			// just "digestSize". Therefore here we just verify with default salt length.
 			err = rsa.VerifyPSS(signer.Public().(*rsa.PublicKey), k.opts.HashFunc(), digest[:], sig, nil)
```

Comment:
What's the reason we don't use the hash function? `&rsa.PSSOptions{Hash: k.opts.HashFunc()}` or `&rsa.PSSOptions{SaltLength: PSSSaltLengthAuto, Hash: k.opts.HashFunc()}` to be explicit. This allows us to check against the expected HashFunc for the test.

Reply:
Seems like the `Hash` in `rsa.PSSOptions` is not used in `rsa.VerifyPSS()`. And passing PSSSaltLengthAuto (0) has the same effect as passing the nil PSSOptions ptr. So I fell they are testing the same there, what do you think?

Reply:
make sense to me. let's keep it as is